protos/route.proto

```diff
@@ -303,6 +330,9 @@ service FlowService {
   rpc MirrorStatus(MirrorStatusRequest) returns (MirrorStatusResponse) {
     option (google.api.http) = { get: "/v1/mirrors/{flow_job_name}" };
   }
+  rpc CDCMirrorStatus(MirrorStatusRequest) returns (CDCStatusResponse) {
```

Comment:
this is unnecessary, fold additional parameters into existing `MirrorStatus` route

Reply:
MirrorStatus route includes peer configuration information, mirror configuration information etc (UI needs) in its response which isn't needed for a status endpoint, and probably not ideal to send this info over the air from a public endpoint. It would also be slower. cc @iskakaushik

Reply:
no route is 'public', all routes should either be unexposed and/or authenticated
you can add fields to the request objects to return less parameters if needed, but 2 routes for similar purposes will make our API unwieldly

Reply:
+1 to consolidating endpoint, long term we should be sending as little peer config info back & forth